pdk/1.0/modules/ROOT/pages/policies-pdk-configure-features-caching.adoc

```diff
@@ -0,0 +1,46 @@
+= Sharing Data Between Workers and Caching
```

Comment:
```suggestion
= Sharing Data Between Workers and the Caching Mechanism
```
Or something. Not sure. At first read I thought that this tutorial included steps for workers to share data amongst themselves, **and** steps for some kind of caching. The wording is a little ambiguous and therefore it confused me.

Or maybe:
"Sharing Worker Data with the Cache"?

Or maybe:
"Caching Worker Data"?

Again, not sure. Something to think about.

Reply:
https://github.com/mulesoft/docs-gateway/pull/504